db/app.js

```diff
@@ -11,6 +12,12 @@ app.get('/api', (req, res) => {
     res.status(200).send(endpoints)
   })
 
+app.get('/api/articles/:article_id', getArticleById)
+
+app.all('*', (_,res) => {
```

Comment:
Really good to include this generic bad path error handler, but would be good to add a test for it too.